fmriprep/cli/run.py

```diff
@@ -126,35 +118,34 @@ def get_parser():
              ' - fsaverage*: FreeSurfer average meshes'
     )
     g_conf.add_argument(
+        '--force-bbr', action='store_true', dest='use_bbr', default=None,
+        help='Always use boundary-based registration (no goodness-of-fit checks)')
+    g_conf.add_argument(
+        '--force-no-bbr', action='store_false', dest='use_bbr', default=None,
+        help='Do not use boundary-based registration (no goodness-of-fit checks)')
+    g_conf.add_argument(
         '--template', required=False, action='store',
         choices=['MNI152NLin2009cAsym'], default='MNI152NLin2009cAsym',
         help='volume template space (default: MNI152NLin2009cAsym)')
-
     g_conf.add_argument(
         '--output-grid-reference', required=False, action='store', default=None,
         help='Grid reference image for resampling BOLD files to volume template space. '
              'It determines the field of view and resolution of the output images, '
              'but is not used in normalization.')
     g_conf.add_argument(
         '--medial-surface-nan', required=False, action='store', default=False,
-        help='Replace medial wall values with NaNs on functional FIFTI files. Only '
+        help='Replace medial wall values with NaNs on functional GIFTI files. Only '
         'performed for GIFTI files mapped to a freesurfer subject (fsaverage or fsnative).')
 
     # ICA_AROMA options
     g_aroma = parser.add_argument_group('Specific options for running ICA_AROMA')
     g_aroma.add_argument('--use-aroma', action='store_true', default=False,
                          help='add ICA_AROMA to your preprocessing stream')
     #  ANTs options
-
     g_ants = parser.add_argument_group('Specific options for ANTs registrations')
-    g_ants.add_argument('--skull-strip-ants', dest="skull_strip_ants", action='store_true',
-                        help='use ANTs-based skull-stripping (default, slow))')
-    g_ants.add_argument('--no-skull-strip-ants', dest="skull_strip_ants", action='store_false',
-                        help="don't use ANTs-based skull-stripping (use  AFNI instead, fast)")
-    g_ants.add_argument('--skull-strip-template', dest="skull_strip_template", action='store',
-                        choices = ['oasis', 'scsnl'], default='scsnl',
-                        help="template to use for skull stripping (default: scsnl)")
-    g_ants.set_defaults(skull_strip_ants=False, skull_strip_template='scsnl')
+    g_ants.add_argument('--skull-strip-template', action='store', default='OASIS',
+                        choices=['OASIS', 'NKI'],
+                        help='select ANTs skull-stripping template (default: OASIS))')
```

Comment:
Here, you had a `scnsl` template, so we've apparently been working in parallel with your lab. This would be a great candidate for another PR, but I've reverted it to our version for this PR.